src/borodinOleh/hw1/High.java

```diff
@@ -0,0 +1,31 @@
+package homework.hw1;
+
+import java.util.Scanner;
+
+public class High {
+    static int input = getNumber();
+
+    public static int getNumber(){
+        System.out.println("Write number");
+        Scanner in = new Scanner(System.in);
+        int result = in.nextInt();
+        return result;
+
+    }
+
+    public static void palindrome(){
+        String str = String.valueOf(input), reverseStr = "";
```

Comment:
I would better write 
`String str = String.valueOf(input);
  String reverseStr = ""; `

No big difference, but easy to find `reverseStr ` :)